app/models/future/policy.rb

```diff
@@ -32,8 +33,22 @@ def slug
       @slug ||= base_path.split('/').last
     end
 
+    def parent_policies
```

Comment:
I think it would make more sense to call this `policy_areas` to use the terminology used by policy-publisher and in the schema/content-items. Both here and elsewhere in the changes.